src/borg/cache.py

```diff
@@ -219,6 +219,10 @@ def __str__(self):
 Chunk index:    {0.total_unique_chunks:20d} {0.total_chunks:20d}"""
         return fmt.format(self.format_tuple())
 
+    def get_summary(self):
+        Summary = namedtuple('Summary', ['total_size', 'total_csize', 'unique_size', 'unique_csize', 'total_unique_chunks', 'total_chunks'])
+        return Summary(*self.chunks.summarize())._asdict()
+
     def format_tuple(self):
         # XXX: this should really be moved down to `hashindex.pyx`
         Summary = namedtuple('Summary', ['total_size', 'total_csize', 'unique_size', 'unique_csize', 'total_unique_chunks', 'total_chunks'])
```

Comment:
Some duplication here, can you make format_tuple() use get_summary for the first two lines? They're identical as far as I see.